src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -6,54 +6,109 @@
+                    new Email("lidavid@example.com"),
+                    new Address("Blk 436 Serangoon Gardens Street 26, #16-43"),
+                    new Nric("S1837493D"), new Age("31"), getTagSet("active")),
+            new Volunteer(new Name("Irfan Ibrahim"), new Phone("92492021"),
+                    new Email("irfan@example.com"), new Address("Blk 47 Tampines Street 20, #17-35"),
+                    new Nric("T0349743E"), new Age("20"), getTagSet("energetic")),
+            new Volunteer(new Name("Roy Balakrishnan"), new Phone("92624417"),
+                    new Email("royb@example.com"),
+                    new Address("Blk 45 Aljunied Street 85, #11-31"),
+                    new Nric("S7692754J"), new Age("40"), getTagSet("eager"))
+        };
+    }
+
+    /**
+     * Returns a list of sample elderly data.
+     *
+     * @return Sample elderly list.
+     */
+    public static Elderly[] getSampleElderly() {
+        return new Elderly[] {
+            new Elderly(new Name("Alex Yeoh"), new Phone("87438807"),
+                    new Email("alexyeoh@example.com"),
+                    new Address("Blk 30 Geylang Street 29, #06-40"),
+                    new Nric("S4392759D"), new Age("29"), new RiskLevel("LOW"),
+                    getTagSet("energetic")),
+            new Elderly(new Name("Bernice Yu"), new Phone("99272758"),
+                    new Email("berniceyu@example.com"),
+                    new Address("Blk 30 Lorong 3 Serangoon Gardens, #07-18"),
+                    new Nric("S9375439F"), new Age("36"), new RiskLevel("LOW"),
+                    getTagSet("colleagues", "friends")),
+            new Elderly(new Name("Charlotte Oliveiro"), new Phone("93210283"),
+                    new Email("charlotte@example.com"),
+                    new Address("Blk 11 Ang Mo Kio Street 74, #11-04"),
+                    new Nric("T9451849S"), new Age("22"), new RiskLevel("LOW"),
+                    getTagSet("neighbours")),
+            new Elderly(new Name("David Li"), new Phone("91031282"),
+                    new Email("lidavid@example.com"),
+                    new Address("Blk 436 Serangoon Gardens Street 26, #16-43"),
+                    new Nric("S1837499D"), new Age("31"), new RiskLevel("LOW"),
+                    getTagSet("family")),
+            new Elderly(new Name("Irfan Ibrahim"), new Phone("92492021"),
+                    new Email("irfan@example.com"), new Address("Blk 47 Tampines Street 20, #17-35"),
+                    new Nric("T0349749E"), new Age("20"), new RiskLevel("LOW"),
+                    getTagSet("classmates")),
+            new Elderly(new Name("Roy Balakrishnan"), new Phone("92624417"),
+                    new Email("royb@example.com"),
+                    new Address("Blk 45 Aljunied Street 85, #11-31"),
+                    new Nric("S7692759J"), new Age("40"), new RiskLevel("LOW"),
+                    getTagSet("colleagues"))
```

Comment:
For the change of sample data here, maybe we should not using the exact same copy of data for Elderly and Volunteer? Because this copy of data will later be used for unit testing, and exactly same NRIC and name could potentially cause some issues

Reply:
Interestingly, since these entities are now stored in separate files, no such cross file validation is currently being done 🤔

Reply:
Theres a TODO below to change the data in the future